test/stdlib/json/JSON_test.rb

```diff
@@ -1,6 +1,27 @@
 require_relative "../test_helper"
 require "json"
 
+
+class JsonToStr
```

Comment:
the JSON library uses a lot of `Object` methods, so it's a lot easier to just make new, custom `.to_str` and `.to_s` classes. When I get around to `JSON` i'll rework a lot of the logic.

Reply:
Defined these two classes in here instead of `test_helper`. In the future the rest of the json test helper classes should be moved over, and more generally helper classes should be local to their files IMO